src/resmom/mom_main.c

```diff
@@ -150,14 +147,6 @@
  * with a pointer to a shared mutex.
  */
 
-#if IRIX6_CPUSET == 1
-#include "pbs_mutex.h"
```

Comment:
Nothing includes pbs_mutex.h any longer. It should be removed from the source tree.